### ISSUE TITLE
WebAIM - CSS in Action: Invisible Content Just for Screen Reader Users

### ISSUE THREAD

**Issue by fititnt** [webiwg/acessibilidade-web-pt#16]
**Link do original**: 
- http://webaim.org/techniques/css/invisiblecontent/

**Rascunho de Trabalho**
- http://acessibilidade.pt.webiwg.org/webaim/tecnicas/css/conteudo-invisivel-apenas-para-leitores-de-tela/

---

**Descrição**: _There are occasional instances where content should be made available to screen reader users, but hidden from sighted users. In most cases, if content (particularly content that provides functionality or interactivity) is important enough to provide to screen reader users, it should probably be made available to all users. _
**Existe traduções em português**: não
## Ações
- [x] Definir pertinência
- [x] Descobrir se é necessário autorização prévia para traduzir
- [x] Caso seja necessário, solicitar autorização para traduzir
- [ ] Adicionar versão original ao repositório
- [ ] Definir uma ou mais pessoas para traduzir original
- [ ] Completar tradução para português
- [ ] Revisar tradução feita

**Comment by fititnt** [comment]
A versão candidata para lançamento final foi publicada em http://acessibilidade.pt.webiwg.org/webaim/tecnicas/css/conteudo-invisivel-apenas-para-leitores-de-tela/. Eu vou ficar em contato direto com o revisor convidado da tradução.

**Comment by fititnt** [comment]
@Dkmister Fechou esse aqui. Você pode enviar e-mail para WebAIM informando da liberação da tradução e tão logo eles dêem resposta, você fecha esse tópico?
